### DISCUSSION TITLE
ElectricShock makes you vomit all over the ship

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14580] [Unstable feedback]
The new affliction ElectricShockWearoff (caused by electricshock over 90) causes Nausea, and if you fail 2 checks in quick succession you'll be vomiting all over the ship.

This is on top of the many things that make you vomit already. 

Fail to repair a JB? Vomit
Too much opium? Vomit
Drink alcohol? Vomit
Chloral Hydrate? Vomit
Watcher? Vomit

Please no nausea.

**Comment by SomeRandomNoobKekeke** [comment]
![raptor bane](https://github.com/user-attachments/assets/b8221a8e-916e-44e3-b6d8-3d5d4fc00bfd)
